channels/serializers.py

```diff
@@ -124,6 +124,7 @@ class PostSerializer(serializers.Serializer):
     title = serializers.CharField()
     upvoted = WriteableSerializerMethodField()
     removed = WriteableSerializerMethodField()
+    stickied = WriteableSerializerMethodField()
```

Comment:
This could just be a `serializers.CharField()` to avoid the method.

Reply:
I'm not exactly sure what you mean  - what method will this let me avoid?

Reply:
I tried that real quick and I get `"False"` instead of `false` back from the API, so it seems like it's coercing the boolean to a string - is there a way around that?

Reply:
:woman_facepalming: I meant to write serializers.BooleanField

Reply:
cool, the BooleanField works great